locales/en-US/app.ftl

```diff
@@ -156,6 +156,11 @@ experimentGradReportReady=We have prepared a full graduation report.
 experimentGoToLink = Go to { $title }
 startedDateLabel = Experiment Start Date: <b>{$startedDate}</b>
 
+// news updates dialog.
+[[newsUpdatesDialog]]
+nonExperimentDialogHeaderLink = Test Pilot
```

Comment:
One more thing occurred to me about these strings: 

Could we reuse [`siteName`](https://github.com/meandavejustice/testpilot/blob/8e644f92c360e753dea5d072b0399b54801ab32c/locales/en-US/app.ftl#L1) (i.e. "Firefox Test Pilot") and maybe [`experimentCardLearnMore`](https://github.com/meandavejustice/testpilot/blob/8e644f92c360e753dea5d072b0399b54801ab32c/locales/en-US/app.ftl#L104) (i.e. "Learn More"). Might save a couple of semi-redundant extra strings.

Maybe not, since the experiment card learn more is not exactly the same purpose as this one, hmm...